examples/resources/grafana_cloud_plugin/resource.tf

```diff
@@ -0,0 +1,5 @@
+resource "grafana_cloud_plugin_installation" "test" {
```

Comment:
You should also place an `import.sh` script beside this file, and it will also show up in the docs